docs/src/man/basics.md

```diff
@@ -52,6 +52,37 @@ The most fundamental type provided by DataFrames.jl is `DataFrame`, where
 typically each row is interpreted as an observation and each column as a
 feature.
 
+!!! note
+
+    DataFrames.jl uses precompilation to improve its responsiveness. However,
+    in some scenarios users might want to avoid precompilaion to improve
+    package installation time and load time. To disable precompilation of
+    DataFrames.jl in your current project you need to install the
+    [SnoopPrecompile.jl](https://github.com/timholy/SnoopCompile.jl/tree/master/SnoopPrecompile)
+    package and then run the following code:
+    ```
+    using SnoopPrecompile
+    SnoopPrecompile.Preferences.set_preferences!(SnoopPrecompile,
+        "skip_precompile" =>
+        union(SnoopPrecompile.Preferences.load_preference(SnoopPrecompile,
+                                                          "skip_precompile",
+                                                          String[]),
+              ["DataFrames"]);
+        force=true)
```

Comment:
Interesting, I haven't played enough with this to realize when `force=true` is necessary.